### ISSUE TITLE
AttributeError: 'super' object has no attribute 'on_list_item__child_clicked'

### ISSUE THREAD

**Issue by davetapley** [mitosch/textual-select#5]
Getting ⬇️ since `v0.21.0`
```
AttributeError: 'super' object has no attribute 'on_list_item__child_clicked'
```

Because:

- https://github.com/Textualize/textual/issues/2324

---

Hack would be to add `_` prefix to call, but perhaps need a broader conversation upstream? 🤔

**Comment by willmcgugan** [comment]
It is probably not necessary to call `super()` here. Textual will call message handlers on base classes by default.

**Comment by mitosch** [comment]
Thank you for the issue @davetapley , I will look into it and try to deliver a fix asap.

And I will mention in the README to use the native select. Thanks for your great work @willmcgugan

**Comment by mitosch** [comment]
Fixed with version 0.3.4. I had to use `_` as a quickfix. I could not find another way by clicking on an element to select and close the dropdown.

In any case, this plugin will slowly disappear because Textual has it's own native implementation of Select.
